src/ts/uiconfig.ts

```diff
@@ -24,6 +24,10 @@ export interface TimelineMarker {
    * Will be rendered in the {@link SeekBarLabel} attached to a {@link SeekBar}.
    */
   title?: string;
+  /**
+   * Url to timeline marker image
+   */
+  imageUrl?: string;
```

Comment:
Can we add a description of what a prefered image resolution is?
![Screenshot 2020-07-29 at 15 06 03](https://user-images.githubusercontent.com/6216959/88803626-0a440a00-d1ad-11ea-99df-23355a800888.png)

Reply:
I can extend it to something like 
```
  /**
   * Url to timeline marker image
   * Prefered small image sizes (~50px)
   * Best performing with SVG format
   */
```